src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.dao;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionDao {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
```

Comment:
```suggestion
    Optional<MovieSession> get(Long id);
```

lets use Optional here